### ISSUE TITLE
Error while manipulating datetime component (using playwright) after updated to CodeceptJS 3.2.1

### ISSUE THREAD

**Issue by andreyagra** [codeceptjs/CodeceptJS#3130]
Important point: within CodeceptJS 3.2.0 version is working properly

```bash
page.$$: Execution context was destroyed, most likely because of a navigation.
at findElements (node_modules/codeceptjs/lib/helper/Playwright.js:2351:18)
at Playwright._locate (node_modules/codeceptjs/lib/helper/Playwright.js:928:12)
at async Playwright.grabNumberOfVisibleElements (node_modules/codeceptjs/lib/helper/Playwright.js:1405:15)
```

> Provide test source code

```js

  async fillDispatchDate(nextMonth=true, decrease=false){
    let dispatchDate = await I.grabValueFrom('#sales_order_date_required');
    dispatchDate = dispatchDate.toString().split("-");
    I.say(dispatchDate);
    let newDispatchDate = new Date(+dispatchDate[2],dispatchDate[1]-1, +dispatchDate[0]);
    I.say(newDispatchDate);

    if (nextMonth) newDispatchDate.setMonth(newDispatchDate.getMonth()+1);
    else {
      const numberOfDays = decrease ? -1 : 1;
      newDispatchDate.setDate(newDispatchDate.getDate() + numberOfDays);
    }
    I.click('#sales_order_date_required');
    I.pressKey('Escape');
    const dateToSetUp = `${newDispatchDate.getDate()}-${newDispatchDate.getMonth()+1}-${newDispatchDate.getFullYear()}`
    I.fillField('#sales_order_date_required', dateToSetUp);
    I.pressKey('Enter');
  },

  async createOrder(){
    let reduceDate = async function () {
      const messageLocate = locate('li').withText('Date required must be before');
      return await I.grabNumberOfVisibleElements(messageLocate)
    }

    let forbiddenDate = async function () {
      const messageLocate = locate('span').withText('Company is closed on the selected date, please choose another');
      const errorMessage = await I.grabNumberOfVisibleElements(messageLocate);
      return errorMessage > 0
    }
    await this.fillDispatchDate();
    let shouldReduceDays = await reduceDate();
    I.say(`Should reduce? ${shouldReduceDays > 0}`);

    while (await forbiddenDate() || shouldReduceDays > 0){
      await this.fillDispatchDate(false, (shouldReduceDays > 0));
      shouldReduceDays = await reduceDate();
    }

    I.click(locate('button').withAttr({class: 'btn btn-success btn-lg dropdown-toggle'}));
  },

```

### Details

* CodeceptJS version: 3.2.1
* NodeJS Version: 16.13.0 LTS
* Operating System: Linux (Ubuntu 20.04.3 LTS - Debian 4.19.171-2 - Docker version 20.10.9)
* Playwright

* Configuration file:

```js
exports.config = {
  tests: './tests/*_test.js',
  output: './output',
  helpers: {
    DbHelper: {
      require: `${process.env.NODE_MODULES_PATH}/node_modules/codeceptjs-dbhelper`,
      maxOldGenerationSizeMb: 8192,
    },
    Playwright: {
      url: "http://localhost",
      show: true,
      browser: 'chromium',
      smartWait: 5000,
      windowSize: "1920x1080",
      fullPageScreenshots: true,
      maxOldGenerationSizeMb: 8192,
   },
   Mochawesome:{
      "uniqueScreenshotNames": "true",
      maxOldGenerationSizeMb: 8192,
    },
  },
  include: {
    I: './steps_file.js',
    masters: './pages/masters.js',
    accessScreen: './pages/accessingScreen',
    newOrder: './steps/newOrderSteps.js',
    newItemOrderFill: './pages/itemOrderFill.js',
    pricingBehaviorControl:  './pages/specialPricingBehaviorControl.js'
  },
  bootstrap: null,
  mocha: {
    reporterOptions: {
      stdout: './output/console.log',
      reportDir: './output',
      reportPageTitle: 'Tests E2E',
      reportTitle: 'Tests E2E',
      reportFilename: 'report',
      quiet: true,
      json: false,
      html: true,
      overwrite: true,
      inline: true,
    },
  },
  name: 'Tests End2End',
  plugins: {
    autoDelay:{
      enabled: false
    },
    screenshotOnFail: {
      enabled: true
    },
  }
};
```

**Comment by DavertMik** [comment]
Thanks for the report. There was a dirty hack we used previously for Playwright & Puppeteer. And I thought that this dirty hack is not needed anymore. Unfortunately, it is still required on some issues. This is unintended break so I will restore it on next patch release. Thank you for creating an issue!

**Comment by DavertMik** [comment]
Will be fixed in 3.2.2
